omod/src/main/java/org/openmrs/web/servlet/LoginServlet.java

```diff
@@ -81,8 +82,16 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 		// look up the allowed # of attempts per IP
 		Integer allowedLockoutAttempts = 100;
 		
-		String allowedLockoutAttemptsGP = Context.getAdministrationService().getGlobalProperty(
-		    GP_ALLOWED_LOGIN_ATTEMPTS_PER_IP, "100");
+		String allowedLockoutAttemptsGP = "";
```

Comment:
Why not initialise this with the default value?

Reply:
I updated the PR.